regression/smv/assign/assign1.desc

```diff
@@ -0,0 +1,8 @@
+KNOWNBUG
+assign1.smv
+
+^file .* line 6: variable `x' already assigned.*$
+^EXIT=1$
+^SIGNAL=0$
+--
+--
```

Comment:
It seems there is one test that now works while the others are all `KNOWNBUG`: do those need further work?

Reply:
Yes, will need more work.